tasks/output.yml

```diff
@@ -0,0 +1,30 @@
+- name: Get system subscription status
+  command: subscription-manager status
+  register: oasis_roles_rhsm_subscription
+  changed_when: false
+  failed_when: false
+
+- name: Get system subscribed pools
+  # This always exits 0 whether the system is subscribed or not;
+  # stdout is empty on an unsubbed system.
+  command: subscription-manager list --consumed --pool-only
+  register: oasis_roles_rhsm_pools
+  changed_when: false
+  failed_when: false
+
+- name: Set system repositories default value
+  # We only get a return fact from the rhsm_repository module if the
+  # rhsm_repositories var is set. If that return fact is not defined,
+  # set that repositories value to false to indicate rhsm_repository
+  # was not used on this host.
+  set_fact:
+    oasis_roles_rhsm_repositories:
```

Comment:
Is this documented somewhere? I don't see it in the new docs you included.

Reply:
Only one fact is output: https://github.com/oasis-roles/rhsm/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R87